tests/1_challangeExit.js

```diff
@@ -0,0 +1,98 @@
+const { Tx, Input, Output, Outpoint, Period, helpers } = require('leap-core');
```

Comment:
typo in the filename `1_challangeExit.js` → `1_challengeExit.js`

Reply:
Cool